### DISCUSSION TITLE
Cannot find github secrets

### DISCUSSION THREAD

**Discussion by v1b3m** [community/community#24204] [Other Feature Feedback, Questions, & Ideas]
<p>I can’t seem to locate where secrets is in the settings tab of my repo(no tab named secrets exists as I expected it to), also appending secrets directly to my repo’s url redirects to a 404. I am stuck. </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/b/bf28dc499b5aa2f97093e0a7472de5ac9a38f78f.png" data-download-href="/uploads/short-url/rh4JSPeQ6qycbBsZlHuXxoeL4dN.png?dl=1" title="Screenshot from 2019-11-10 17-13-52.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/39791369/181083474-adf7c549-ef8b-4f98-8673-81de2be8c99a.png" alt="Screenshot from 2019-11-10 17-13-52.png" data-base62-sha1="rh4JSPeQ6qycbBsZlHuXxoeL4dN" width="690" height="387" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/b/bf28dc499b5aa2f97093e0a7472de5ac9a38f78f_2_690x387.png, https://aws1.discourse-cdn.com/github/optimized/2X/b/bf28dc499b5aa2f97093e0a7472de5ac9a38f78f_2_1035x580.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/b/bf28dc499b5aa2f97093e0a7472de5ac9a38f78f.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/b/bf28dc499b5aa2f97093e0a7472de5ac9a38f78f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot from 2019-11-10 17-13-52.png</span><span class="informations">1366×768 247 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by hefedev** [comment]
<p>Hey There,</p>
<p>The option to generate a secret is located under the developer tab and inside the app profile page. This can be either OAuth, Github-app.</p>
<p>Kind Regards,</p>

**Comment by brianamarie** [comment, marked as answer]
<p>Hi <a class="mention" href="/v1b3m">@v1b3m</a>. Can you please confirm that you have opted in to the<a href="https://github.com/features/actions" rel="nofollow noopener">Actions beta</a>? If you haven’t, that may be preventing you from seeing the <code>secrets</code> part of the settings.</p>

**Comment by v1b3m** [comment]
<p>Hey, <a class="mention" href="/brianamarie">@brianamarie</a> thanks so much for the advice, I had not opted in?</p>

**Comment by v1b3m** [comment]
<p>Hey, <a class="mention" href="/brianamarie">@brianamarie</a> thanks so much, I had not opted in.</p>

**Comment by v1b3m** [comment]
<p>I am sorry for being a pain, but there is nothing on the page that indicates registration / sign up. Only link I am seeing is one to the features in the market place.</p>

**Comment by v1b3m** [comment]
<p>It is now working, thanks so much.</p>

**Comment by v1b3m** [comment]
<p>This is not what I was looking for, thanks though.</p>

**Comment by mrmironex** [comment]
<p>Мдя… Вот это поворот…</p>
